README.md

```diff
@@ -321,6 +331,15 @@ client.getUnspentTransactionOutputs([{
 }], { extension: 'json' }, [callback])
 ```
 
+### command(command, [arguments...])
+Allows you to directly call bitcoind's [RPC methods](https://bitcoin.org/en/developer-reference#remote-procedure-calls-rpcs).
```

Comment:
This is already mentioned above under RPC > Methods. All REST methods will be exposed individually so I don't think it makes sense to document `command()` for calling REST methods, as it can be counterproductive.

Reply:
Mentioning something isn't the same as documenting it. Until literally *all* REST methods are exposed, `command` will remain necessary and should be documented. I'd recommend only removing it from the docs when there's absolutely nothing you'd want to do that requires it.

Reply:
It won't work @fresheneesz. The `command` method is bound to the RPC request path and JSON-RPC interface.